en/oop.md

```diff
@@ -43,6 +43,42 @@ The following class modifiers are supported:
     
     }
 
+<a name='classes-interfaces'></a>
+### Implementing Interfaces
+Zephir classes can implement any number of interfaces, provided that these interfaces are visible for the extension to use. However there are times that your extension might require to implement an interface that is built in another extension or package. 
+
+If we want to implement the `MiddlewareInterface` from the `PSR` package, we will need to:
```

Comment:
Should probably clarify that we need the PSR extension, specifically, as normal Composer packages won't be available to extensions at load time.

Reply:
I have a note below stating just that. Do I need to add it here too you think?